Source/Scene/PointCloudEyeDomeLighting.js

```diff
@@ -0,0 +1,400 @@
+define([
+        '../Core/Cartesian2',
+        '../Core/Math',
+        '../Core/clone',
+        '../Core/Color',
+        '../Core/combine',
+        '../Core/ComponentDatatype',
+        '../Core/defined',
+        '../Core/destroyObject',
+        '../Core/FeatureDetection',
+        '../Core/Geometry',
+        '../Core/GeometryAttribute',
+        '../Core/PixelFormat',
+        '../Core/PrimitiveType',
+        '../Renderer/BufferUsage',
+        '../Renderer/ClearCommand',
+        '../Renderer/DrawCommand',
+        '../Renderer/Framebuffer',
+        '../Renderer/Pass',
+        '../Renderer/PixelDatatype',
+        '../Renderer/RenderState',
+        '../Renderer/Sampler',
+        '../Renderer/ShaderSource',
+        '../Renderer/ShaderProgram',
+        '../Renderer/Texture',
+        '../Renderer/TextureMagnificationFilter',
+        '../Renderer/TextureMinificationFilter',
+        '../Renderer/TextureWrap',
+        '../Renderer/VertexArray',
+        '../Scene/BlendEquation',
+        '../Scene/BlendFunction',
+        '../Scene/BlendingState',
+        '../Scene/StencilFunction',
+        '../Scene/StencilOperation',
+        '../Shaders/PostProcessFilters/PointCloudEyeDomeLighting'
+    ], function(
+        Cartesian2,
+        CesiumMath,
+        clone,
+        Color,
+        combine,
+        ComponentDatatype,
+        defined,
+        destroyObject,
+        FeatureDetection,
+        Geometry,
+        GeometryAttribute,
+        PixelFormat,
+        PrimitiveType,
+        BufferUsage,
+        ClearCommand,
+        DrawCommand,
+        Framebuffer,
+        Pass,
+        PixelDatatype,
+        RenderState,
+        Sampler,
+        ShaderSource,
+        ShaderProgram,
+        Texture,
+        TextureMagnificationFilter,
+        TextureMinificationFilter,
+        TextureWrap,
+        VertexArray,
+        BlendEquation,
+        BlendFunction,
+        BlendingState,
+        StencilFunction,
+        StencilOperation,
+        PointCloudEyeDomeLightingShader
+    ) {
+    'use strict';
+
+    /**
+     * Eye dome lighting.
+     * Requires support for EXT_frag_depth, OES_texture_float, and WEBGL_draw_buffers extensions in WebGL 1.0.
+     *
+     * @private
+     */
+    function PointCloudEyeDomeLighting() {
```

Comment:
@lilleyse @bagnell what is the plan for this in the context of the upcoming post processing framework?

Merge as is for now and then update to use the framework?  Always keep as is?

Reply:
I would merge as is. It looks like it would be simple to update to use the framework later. Most of the work here seems to be creating/executing derived commands for point clouds.

Reply:
OK, @likangning93 please submit an issue to remind us to do this later.

Reply:
Was this issue submitted?